cherrypy/tutorial/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM python:3.6-alpine
```

Comment:
Better would be `python:3` or maybe `python:3-alpine` (if such a thing exists), for future compatibility.